src/test/kotlin/com/exactpro/th2/common/util/TestMultiMapDeserializer.kt

```diff
@@ -32,23 +34,38 @@ class TestMultiMapDeserializer {
         private val OBJECT_MAPPER: ObjectMapper = ObjectMapper()
 
         init {
-            OBJECT_MAPPER.registerModule(KotlinModule())
+            OBJECT_MAPPER.registerModule(
+                KotlinModule.Builder()
+                    .withReflectionCacheSize(512)
+                    .configure(KotlinFeature.NullToEmptyCollection, false)
+                    .configure(KotlinFeature.NullToEmptyMap, false)
+                    .configure(KotlinFeature.NullIsSameAsDefault, false)
+                    .configure(KotlinFeature.SingletonSupport, false)
```

Comment:
issue (non-blocking): This might result it several instances of Kotlin `object` that is not correct. We should set enable this feature. Even if this is just in test